bitcoin/src/util/mod.rs

```diff
@@ -97,6 +96,34 @@ pub use crate::bip32;
 #[deprecated(since = "0.30.0", note = "Please use crate::bip158")]
 pub use crate::bip158;
 
+/// The `hash` module was renamed to `../merkle_tree`.
```

Comment:
Although GH says "renamed" and we think of it as renaming, a more sensible representation is "removed and the merkle tree functions were moved to `merkle_tree`"

Reply:
Noted, all thees "deprecated" messages should go away in a version or two so unless I re-spin for some other reason I'll leave this as is.

Reply:
Changed to: `/// Functions from the `hash` module were renamed and moved to `../merkle_tree`.`